shared/chat/conversation/input/container.js

```diff
@@ -82,10 +82,15 @@ export default compose(
         this.props.inputFocus()
       }
     },
+    componentWillUnmount: function () {
+      this.props.onStoreInputText(this.props.inputValue())
```

Comment:
mobile unmounts and doesn't really switch convos due to it being on different pages